packer/scripts/install-ruby-gcp.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -x
+
+sudo -u deploy bash <<EOS
+  gpg2 --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+  curl -sSL https://get.rvm.io | bash -s stable
+
+  source /home/deploy/.rvm/scripts/rvm
+  cd /home/deploy
+  rvm install --quiet-curl 2.6.1
```

Comment:
We should try to have the version updated here

Reply:
For the ruby version update, I take the ruby version from `.ruby-version` file in `opendax`